src/scripts/clipperUI/components/annotationInput.tsx

```diff
@@ -102,7 +102,8 @@ class AnnotationInputClass extends ComponentBase<AnnotationInputState, ClipperSt
 						aria-label={Localization.getLocalizedString("WebClipper.Accessibility.ScreenReader.InputBoxToChangeTitleOfOneNotePage")}
 						style={Localization.getFontFamilyAsStyle(Localization.FontFamily.Regular)}
 						value={!!this.props.clipperState.previewGlobalInfo.annotation ? this.props.clipperState.previewGlobalInfo.annotation : ""}
-						onblur={this.onDoneEditing.bind(this)} {...this.onElementFirstDraw(this.setFocus)}>
+						onblur={this.onDoneEditing.bind(this) } { ...this.onElementFirstDraw(this.setFocus)}
+						placeholder="Jorge was here 2!" >
```

Comment:
Remove dummy placeholder?

Reply:
This shouldn't be here, good catch

Reply:
Turns out I didn't hit save Addressed now.